app/routes/circulars._archive._index/CircularsIndex.tsx

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Link } from '@remix-run/react'
+
+import type { CircularMetadata } from '../circulars/circulars.lib'
+
+export default function ({
+  allItems,
+  searchString,
+  totalItems,
+  query,
+}: {
+  allItems: CircularMetadata[]
+  searchString: string
+  totalItems: number
+  query?: string
+}) {
+  return (
+    <>
+      {query && (
+        <h3>
+          {totalItems} result{totalItems != 1 && 's'} found.
+        </h3>
+      )}
```

Comment:
Probably better for this to be part of route.tsx. This is the only part of the component that depends directly on `query`.